spec.html

```diff
@@ -10629,6 +11002,17 @@ <h1>ECMAScript Function Objects</h1>
             The Environment Record that the function was closed over. Used as the outer environment when evaluating the code of the function.
           </td>
         </tr>
+        <tr>
+          <td>
+            [[PrivateEnvironment]]
+          </td>
+          <td>
+            PrivateEnvironment Record | *null*
+          </td>
+          <td>
+            The PrivateEnvironment Record for Private Names that the function was closed over. *null* if this function is not syntactically contained within a class. Used as the outer PrivateEnvironment for inner classes when evaluating the code of the function.
```

Comment:
"closed over" is a bit of an implementation term. Maybe "can reference"?

Reply:
"closed over" is the term already used for Environment Records:

> The Environment Record that the function was closed over.

This just copies that wording for PrivateEnvironment Records.

Reply:
Okay, wfm.